packages/widget/src/AppDrawer.style.tsx

```diff
@@ -96,3 +96,24 @@ export const DrawerButtonTypography = styled(Typography)(({ theme }) => ({
   transform: 'rotateZ(180deg)',
   writingMode: 'vertical-rl',
 }));
+
+export const CloseButtonLayout = styled(Box)(
```

Comment:
There is an IconButton component from MUI, it would be better to use it instead. Also, we don't have bordered icons anywhere in the widget. 🙂

Reply:
Done!